modules/primer-marketing-buttons/lib/button.scss

```diff
@@ -0,0 +1,30 @@
+.btn-orange { @include btn-solid($white, lighten($orange-500, 7.5%), darken($orange-500, 7.5%)); }
+.btn-blurple { @include btn-solid($white, mix($blue-400, $purple-400, 35%), mix($blue-600, $purple-600, 35%)); }
+
+.btn-outline-purple { @include btn-outline($purple); }
+.btn-outline-orange { @include btn-outline($orange); }
+.btn-outline-green { @include btn-outline($green); }
+
+@mixin btn-transparent-active {
+  color: #333;
+  background-color: #fff;
+  background-image: none;
+  border-color: #fff;
+}
+
+.btn-transparent {
+  padding: 0.55em 1em;
+  font-size: inherit;
+  font-weight: 500; // to avoid faux-bolding
```

Comment:
I'm curious about what "faux-bolding" means, we've updated all our font-weights to bold (600), or normal (400) in most other places. If we do need it, you can use the `$font-weight-semibold` variable instead.

Reply:
i also just remembered a few of these styles from github/github where i copied them over from were literally declared specifically for where a `.btn-transparent` was used on the marketing homepage. ive removed the font-weight, font-size, padding, line-height, and border-radius as they should be inherited from `.btn` or the other btn size classes, e.g.`.btn-large`